doc/man3/SSL_read.pod

```diff
@@ -51,40 +51,47 @@ call. If B<num> is higher than the number of bytes buffered then the read
 functions will return with the bytes buffered. If no more bytes are in the
 buffer, the read functions will trigger the processing of the next record.
 Only when the record has been received and processed completely will the read
-functions return reporting success. At most the contents of the record will
+functions return reporting success. At most the contents of one record will
 be returned. As the size of an SSL/TLS record may exceed the maximum packet size
 of the underlying transport (e.g. TCP), it may be necessary to read several
 packets from the transport layer before the record is complete and the read call
 can succeed.
 
+If B<SSL_MODE_AUTO_RETRY> is not set and a non-application data record has been
```

Comment:
You might want to instead say "If SSL_MODE_AUTO_RETRY has been switched off...".